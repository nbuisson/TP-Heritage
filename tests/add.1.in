R 1 2 3 4
C 1 6 1
PL 1 3 1 4 1 4 1 3
L 1 3 5 9
L -1 4 1 -9
EXIT
