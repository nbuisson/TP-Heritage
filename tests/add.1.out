OK R 1 2 3 4 
OK C 1 6 1
OK PL 1 3 1 4 1 4 1 3
OK L 1 3 5 9
OK L -1 4 1 -9
5
